app/templates/views/roadmap.html

```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Let services upload their own logo for emails and letters</li>
     <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
-    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
-    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our database and applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
+    <li>Add multilingual letter templates and attachments to letters so services can provide information in several languages</li>
+    <li>Improve our email functionality</li>
+    <li>Making it easier for services to go-live</li>
```

Comment:
We tend to talk about a ‘request to go live’ rather than just ‘go live’ or ‘go-live’ in isolation.

I think we could be better about explaining trial mode and live in general – but that’s a problem for another day.

For now I might suggest something like:

```suggestion
    <li>Make it easier for services to send a request to go live</li>
```

Or

```suggestion
    <li>Make it easier for services to remove trial mode restrictions</li>
```

Or a combination of the 2. 

I haven’t been super close to this work, though, so perhaps this doesn’t adequately capture the actual changes we’ve made?